spacelift/resource_module.go

```diff
@@ -247,6 +247,12 @@ func resourceModule() *schema.Resource {
 				Optional:    true,
 				Default:     false,
 			},
+			"public": {
```

Comment:
Maybe we should also set `ForceNew` to `true` so that if someone tries to change it after creation Terraform will indicate the resource will be recreated rather than just silently ignoring the change?

Reply:
didnt know that was a thing! Done!